projects/observability/src/shared/components/entity-renderer/entity-renderer.component.ts

```diff
@@ -59,8 +62,9 @@ export class EntityRendererComponent implements OnChanges {
       this.setIconType();
     }
   }
+
   public onClickNavigate(): void {
-    this.navigable && this.entity && this.entityNavService.navigateToEntity(this.entity);
+    this.navigable && this.entity && this.entityNavService.navigateToEntity(this.entity, this.inactive);
```

Comment:
What does inactive do? why do we need this?

Reply:
If an entity is inactive, it means we haven't seen any spans in the time range. Therefore, some of the screens won't show any data available and we may want to navigate to alternative pages instead for those APIs when drilled in to.

Reply:
So is it same as under discovery entity? 
The name is not clear honestly. `navigationFunction(entityId, sourceRoute, isInactive)` i guess this alternate url would be defined in the entity map?